events-api/.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
Why prettier file was edited?

Reply:
fixed, sorry for that. 
Also, for some reason I always have problems with Postman, so I have gotten pretty used to curl at this point. Anyways thanks for the suggestion)